### ISSUE TITLE
Privacy-respecting statistics on use of OpenOversight

### ISSUE THREAD

**Issue by redshiftzero** [lucyparsons/OpenOversight#161]
We often get questions like "How many people have used OO to file complaints?". We might want to think about privacy-respecting ways to provide an answer here that isn't "no logs no masters". Just food for thought.

**Comment by freddymartinez9** [comment]
The leading candidate here is something like [Piwik](https://piwik.org/) which I know other organizations (FPF, maybe FLM) use for this very reason.

**Comment by jujueyeball** [comment]
Library Freedom Project recommended it to librarians at a workshop last
month too.

--
Daniel Massoglia
Attorney and Counselor at Law
(On A Cell Phone)

On Jan 4, 2017 5:22 PM, "Freddy Martinez" <notifications@github.com> wrote:

The leading candidate here is something like Piwik <https://piwik.org/>
which I know other organizations (FPF, maybe FLM) use for this very reason.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/lucyparsons/OpenOversight/issues/161#issuecomment-270514660>,
or mute the thread

**Comment by fritzdavenport** [comment]
Sounds like it's Matomo now? Looks like it can run in docker?

https://matomo.org/docs/installation/#start-the-installation

https://hub.docker.com/_/matomo

**Comment by b-meson** [comment]
Yep the project definitely got renamed. We have thought about it but I think I stopped looking at it because it would have required PHP (this was a long time ago so my memo might be bad)
